docs_user/modules/openstack-planning.adoc

```diff
@@ -204,6 +204,23 @@ are eligible to run the different OpenStack services.  Refer to the xref:node-se
 selector] to learn more about using labels to define
 placement of the OpenStack services.
 
+== TLS Everywhere
+
+If the Director deployment was deployed with TLS Everywhere, FreeIPA (IDM) is used
+for providing services with certificates. Certmonger is installed on all hosts,
+which provisions the certificates for services running on the host.
+
+The new Operator based deployment uses cert-manager to provision the certificates.
+Instructions will be provided on how to retrieve the root CA from IPA and copy it
+over to the new environment.
```

Comment:
Lets be a little more precise.

If the Director deployment was deployed with TLS Everywhere, FreeIPA (IdM) is used
to issue certificates for the OpenStack services. Certmonger, a client process which is installed on all hosts,
interacts with FreeIPA (IdM) to request, install, track and renew these certificates.

The new Operator based deployment uses the cert-manager operator to issue, track and renew the certificates.